### PR TITLE
Add Mendeley Desktop to list of supported apps

### PR DESCRIPTION
**Pull request by spencerwooo**
Mendeley Desktop (Qt-based)'s font can also be altered by FontMod. Mendeley is a reference management software for academic uses.

For more information, see article here: [使用 FontMod 为 Win32 应用程序自定义字体 | 一日一技](https://sspai.com/post/58542)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
 
 Simple hook tool to change Win32 program font. Works with some GDI or Qt based program.
 
-> Tested works with [Telegram Desktop](https://desktop.telegram.org/) and [Kleopatra (Gpg4Win)](https://www.gpg4win.org/).
+> Proven workable on [Telegram Desktop](https://desktop.telegram.org/), [Kleopatra (Gpg4Win)](https://www.gpg4win.org/) and [Mendeley Desktop](https://www.mendeley.com/download-desktop/).
 
 # Usage
-[Download](https://github.com/ysc3839/FontMod/releases) `FontMod.dll` and rename to `winmm.dll`, then put in the folder of program exe.
```
Comment (ysc3839) [inline README.md]: Double spaces will be converted to line break in Markdown, please don't remove them.

Comment (spencerwooo) [reply README.md]: Sorry, VS Code automatically removed the whitespace. Hold on.

Comment (spencerwooo) [reply README.md]: Restored whitespace in the latest commit, please review.

**Comment by ysc3839** [comment]
Merged, thanks!

**Pull request merged**